internal/pkg/mapper/orchestrator/remote_load.go

```diff
@@ -79,6 +81,27 @@ func (l *remoteLoader) load() error {
 		l.config.Orchestration.Phases = append(l.config.Orchestration.Phases, *phase)
 	}
 
+	// Set paths if parent path is set
+	if l.manifest.Path() != "" {
```

Comment:
`remote load`, pridanie generovania paths ku `phases/tasks` ak uz config existuje.
- pouzije sa existujuca cesta, ak napr. `phase` uz existuje v `local` state, ... alebo sa vygeneruje nova.
- ak `local` objekt neexistuje, tak sa jeho cesta aj cesty ku `phase/task` vygeneruju automaticky, tak preto ta podmienka
- mozno by sa to dalo nejak zovseobecnit, ... pozriem neskor.